.travis.yml

```diff
@@ -1,37 +1,30 @@
 language: python
-
+dist: trusty
 # This (sudo: false) is needed to "run on container-based infrastructure" on
 # which cache: is available
 # http://docs.travis-ci.com/user/workers/container-based-infrastructure/
-sudo: false
-
-addons:
-  apt:
-    packages:
-     - zeroc-ice34
```

Comment:
Can we use `zeroc-ice35` now it is white listed (https://github.com/travis-ci/apt-package-whitelist/pull/3656)?